<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>660</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Citrus Excel Change</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>icon.ico</selectedon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>10</y>
      <width>981</width>
      <height>630</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_read">
     <attribute name="title">
      <string>Прочитать</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>160</y>
        <width>61</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Колонка</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>50</y>
        <width>61</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Лист</string>
      </property>
     </widget>
     <widget class="QComboBox" name="object_columns">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>190</y>
        <width>400</width>
        <height>40</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="object_sheet">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>400</width>
        <height>40</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="object_option_read">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>410</y>
        <width>400</width>
        <height>40</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>380</y>
        <width>101</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Действие</string>
      </property>
     </widget>
     <widget class="QPushButton" name="object_btn_start_read">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>500</y>
        <width>250</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Начать</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_object_search_text">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>230</y>
        <width>421</width>
        <height>141</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPlainTextEdit" name="object_search_text">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>400</width>
         <height>80</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>71</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Поиск</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_change">
     <attribute name="title">
      <string>Редактировать</string>
     </attribute>
     <widget class="QComboBox" name="object_option_change">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>260</y>
        <width>400</width>
        <height>41</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QFrame" name="frame_copy">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPlainTextEdit" name="object_copy">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>71</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>С ячейки</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_past">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>330</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPlainTextEdit" name="object_past">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>71</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>В ячейку</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>230</y>
        <width>101</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Действие</string>
      </property>
     </widget>
     <widget class="QPushButton" name="object_btn_start_change">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>500</y>
        <width>250</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Начать</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_object_search_text_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>421</width>
        <height>211</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>41</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Текст</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="object_search_text_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>400</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>351</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>для разделения разного текста используйте  ::</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="object_after_text">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>400</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>После какого текста вставить</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>для вставки</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>430</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Формат указания ячейки: AC</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_result">
     <attribute name="title">
      <string>Результат</string>
     </attribute>
     <widget class="QPlainTextEdit" name="output_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>981</width>
        <height>601</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
